lib/internal/url.js

```diff
@@ -505,11 +516,21 @@ class URLSearchParams {
 
     const list = this.#searchParams;
     name = toUSVString(name);
+
+    if (value !== undefined) {
+      value = toUSVString(value);
+    }
+
     for (let i = 0; i < list.length; i += 2) {
       if (list[i] === name) {
-        return true;
+        if (value === undefined) {
+          return true;
+        } else if (list[i + 1] === value) {
+          return true;
+        }
```

Comment:
It seems we can reduce this into:

```suggestion
        if (value === undefined || list[i + 1] === value) {
          return true;
        }
```

Reply:
will make this change. thanks.

Reply:
Thanks! I'll approve and request ci when this changed is pushed.